### ISSUE TITLE
Text is cut off when using TtfFont (text2)

### ISSUE THREAD

**Issue by unknownguy2002** [korlibs/korge#294]
<img width="239" alt="Image with text 'This is text!' but a small part of the bottom is cut off" src="https://user-images.githubusercontent.com/24543395/95653109-974dfc80-0b28-11eb-92e2-584278c74ae1.png">
Text that is using text2 (TTF fonts) has a small part of the bottom cut off. This happens for many different types of fonts I have tested it with. This does not happen with text instead of text2 

Thanks!

**Comment by soywiz** [comment]
Can you place the snippet of code you used to create/display it? I guess it is render to a texture as a static text, but just to be sure

**Comment by unknownguy2002** [comment]
```kotlin
// Title
text2("THIS IS TEXT!", font = font,fontSize = 35.0) {
    position(views.virtualWidth / 5, views.virtualHeight / 5)
}```
I hope I'm not doing something incorrectly 😅
Thanks!

**Comment by soywiz** [comment]
No, it is ok. That's a static text, so the problem is when rendering the TTF into an image. Probably will require 1 pixel more at the bottom. For dynamic texts you can generate a bitmapfont from the TTF, and that works differently.

**Comment by soywiz** [comment]
Can you provide the TTF file so I can test it is fixed?

**Comment by unknownguy2002** [comment]
This is a different font from the one shown in the picture but both have issues with the bottom row: https://fonts.google.com/specimen/Red+Hat+Display

Hope this helps!

**Comment by soywiz** [comment]
Should be fixed on 2.0.0.2-rc2, feel free to reopen it if it is not the case
